keps/sig-scheduling/5278-nominated-node-name-for-expectation/kep.yaml

```diff
@@ -0,0 +1,36 @@
+title: Nominated node name for an expected pod placement
+kep-number: 5278
+authors:
+  - "@sanposhiho" 
+  - "@wojtek-t" 
```

Comment:
Put you here @wojtek-t. Maybe we can just move you to PRR reviewer if you don't have many things to edit.

Reply:
@wojtek-t Who do you think we can ask for PRR?

Reply:
I would suggest @soltysh probably
Given that I'm pushing some of my ideas through this KEP, it probably shouldn't be me :)

Reply:
/assign @soltysh 

Please assign it to someone else, if needed! 🙇